brayns/Brayns.cpp

```diff
@@ -120,10 +118,15 @@ struct Brayns::Impl : public PluginAPI
         const auto windowSize =
             _parametersManager.getApplicationParameters().getWindowSize();
 
-        _engine->reshape(windowSize);
+        auto& camera = _engine->getCamera();
+        camera.updateProperty("aspect",
+                              static_cast<double>(windowSize.x()) /
+                                  static_cast<double>(windowSize.y()));
+        for (auto frameBuffer : _frameBuffers)
+            frameBuffer->resize(windowSize);
```

Comment:
that's what the code does, how does the comment give more/different information?

Reply:
For a casual reader, it's not obvious at this point that _frameBuffers is not storing all the frame buffers, but just those that were created by BaseWindow. Spare it if you deemed it unnecessary. I mentioned it because despite it doesn't make sense to store other buffers here, you cannot know it for sure without going to the code where they are assigned, assuming it is just an educated guess, but still a guess.

Reply:
No, it stores the buffers that are created in this file.